### PR TITLE
Updates server associations and adds details

### PR DESCRIPTION
**Pull request by rodneyhbrown7**
@miq-bot add_label: wip, providers/physical-infrastructure

### CODE REVIEW

**Comment by juliancheal** [comment]
@miq-bot add_label wip, providers/physical-infrastructure

File: app/models/manageiq/providers/lenovo/physical_infra_manager/physical_server.rb [hunk 1]
```diff
@@ -3,5 +3,8 @@ class Lenovo::PhysicalInfraManager::PhysicalServer < ::PhysicalServer
     def name
       "physical_server"
     end
+
+    belongs_to :ext_management_system, :foreign_key => "ems_id", :class_name => "ManageIQ::Providers::Lenovo::PhysicalInfraManager"
+    has_one :host, :foreign_key => "service_tag", :primary_key => "serial_number"
```
Comment (juliancheal) [inline app/models/manageiq/providers/lenovo/physical_infra_manager/physical_server.rb]: We don't need the `belongs_to` and `has_one` relationships here as they're defined in the super class `Physical Server`

**Comment by rodneyhbrown7** [comment]
Close and reopen to run the travis tests again

**Comment by rodneyhbrown7** [comment]
Reopen to rerun travis tests

**Comment by rodneyhbrown7** [comment]
@miq-bot remove-label wip

**Pull request merged**